arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/CreateSection.java

```diff
@@ -190,24 +195,24 @@ public void runPostTx(Hub hub, WorldView state, Transaction tx) {
           hub,
           commonImcFragment,
           ContextFragment.readContextData(hub.callStack()),
-          new AccountFragment(oldCreatorSnapshot, newCreatorSnapshot),
+          hub.factories().accountFragment().make(oldCreatorSnapshot, newCreatorSnapshot),
           ContextFragment.nonExecutionEmptyReturnData(hub.callStack()));
     } else if (this.failures.any()) {
       if (creatorReverted) {
         this.addFragmentsWithoutStack(
             hub,
             commonImcFragment,
-            new AccountFragment(oldCreatorSnapshot, newCreatorSnapshot),
-            new AccountFragment(oldCreatedSnapshot, newCreatedSnapshot),
-            new AccountFragment(newCreatorSnapshot, oldCreatorSnapshot),
-            new AccountFragment(newCreatedSnapshot, oldCreatedSnapshot),
+            hub.factories().accountFragment().make(oldCreatorSnapshot, newCreatorSnapshot),
```

Comment:
`hub.factories().accountFragment()` can be extracted into a separate local variable.